test/cpp/test_tensor.cpp

```diff
@@ -10,7 +10,12 @@
 namespace torch_xla {
 namespace cpp_test {
 
-TEST(TensorTest, TestAdd) {
+class TensorTest : public ::testing::Test {
+ protected:
+  static void SetUpTestSuite() { at::manual_seed(42); }
```

Comment:
Also, if I am not mistaken, I think we need to hook SetUp() as we want to seed to be reset at every test.

Reply:
Do we really want that? Doing it once per process lifetime will still make tests deterministic.

Reply:
I think so.
You can have tests which are "for-i-in-rand(): rand()"

Reply:
Sure, but I can't think of a good use case for that. In any case, there's not much harm in setting it for every test, I'll do that instead.